src/shared/components/drop-target.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "react-emotion";
+
+const StyledDropTarget = styled.div`
+  align-content: center;
+  align-items: center;
+  background: #ededed;
+  border-radius: 5px;
+  box-sizing: border-box;
+  display: grid;
+  justify-content: center;
+  justify-items: center;
+  padding: 20px;
+
+  width: ${props => (props.width ? `${props.width}px` : "100%")};
+
+  /* Specify an exact height so that the target doesn't grow or shrink when the
+     hover message is displayed */
+  height: ${props => (props.height ? `${props.height}px` : "100px")};
+
+  &.hovered {
```

Comment:
could we provide some kind of animation on hover?

Reply:
(I can see this done in the story and not the component)

Reply:
Animation has been tricky. Not quite sure why my `transition`s aren't being applied. Something about re-rendering the component, maybe. I'll keep at it.